src/components/CompareResults/RevisionRow.tsx

```diff
@@ -189,6 +236,12 @@ function determineStatus(improvement: boolean, regression: boolean) {
   return '-';
 }
 
+function determineDeltaHintStyle(improvement: boolean, regression: boolean) {
+  if (improvement) return 'improvement';
+  if (regression) return 'regression';
+  return '';
+}
```

Comment:
nit: Would it be possible to make the classes more specific? I'm thinking of `status-hint-improvement` and `status-hint-regression`

Reply:
Ah yeah that looks like a better option. Changed it, thanks!